### DISCUSSION TITLE
HotRod upgrade to OTEL SDK changed the ENV vars.

### DISCUSSION THREAD

**Discussion by filipeversehgi** [jaegertracing/jaeger#4199] [General]
Hey, did this happened with anyone?

I used to set JAEGER_AGENT_HOST and JAEGER_AGENT_PORT to point HotRod to my Jeager ThriftCompact endpoint, but since the switch 2 days ago to OTEL SDK, the only variable working is OTEL_EXPORTER_JAEGER_ENDPOINT, and it's not documented anywhere.

I wanted to help/make a PR, but I'm not sure the best place to add that. (Readme, maybe?)

**Comment by yurishkuro** [comment]
My bad, I should've added this to the changelog as breaking change (it was mentioned in the PR). But perhaps it's worth adding a bit of logic to clone old style env vars into equivalent OTEL env vars (and log a warning).

**Reply by yurishkuro** [reply]
Fixing in https://github.com/jaegertracing/jaeger/pull/4200

**Comment by yurishkuro** [comment]
Also, are you seeing official documentation somewhere that talks about env vars for hotrod?

**Reply by filipeversehgi** [reply]
I used to check the env vars on the Readme from HotRod folder, in that section:
[Run from Docker](https://github.com/jaegertracing/jaeger/tree/main/examples/hotrod#run-hotrod-from-docker)

But that idea of closing the old vars into the new one is also interesting.

**Reply by yurishkuro** [reply]
Thanks - fixing in https://github.com/jaegertracing/jaeger/pull/4200
